commands/publish/index.js

```diff
@@ -84,6 +84,18 @@ class PublishCommand extends Command {
           `
         );
       }
+
+      if (!this.options.skipNpm) {
+        GitUtilities.fetchGitRemote(this.execOpts);
```

Comment:
I think we could simply call the fetch from `isBehindUpstream()`? I don't see much value in separating it, kinda strikes me as an implementation detail of `isBehindUpstream()`.